app/src/main/java/org/wikipedia/games/onthisday/OnThisDayGameArticleBottomSheet.kt

```diff
@@ -157,12 +110,12 @@ class OnThisDayGameArticleBottomSheet(
         if (isSaved) {
             LongPressMenu(view, existsInAnyList = false, callback = object : LongPressMenu.Callback {
                 override fun onAddRequest(entry: HistoryEntry, addToDefault: Boolean) {
-                    ReadingListBehaviorsUtil.addToDefaultList(activity, pageTitle, addToDefault, InvokeSource.ON_THIS_DAY_GAME_ACTIVITY)
+                    ReadingListBehaviorsUtil.addToDefaultList(requireActivity(), pageTitle, addToDefault, InvokeSource.ON_THIS_DAY_GAME_ACTIVITY)
```

Comment:
The snackbar does not show up properly, I assume it got overlapped by the bottomsheet.

Reply:
Great observation, and here's my solution:
I updated `FeedbackUtil` to be able to display a Snackbar _on top of_ a bottom sheet (by _auto-detecting_ if the current activity is displaying a bottom sheet)!